dts/bindings/led/x-powers,axp192-led.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  AXP192 LED controller
+
+  The AXP192 has one LED can automatically display error or charging status,
+  or be controlled by software.
+
+compatible: "x-powers,axp192-led"
+
+on-bus: axp192
+
+properties:
+  x-powers,mode:
+    type: string
+    enum:
+      - "by-reg"
+    description:
+      Select the LED control method.
+      If you select "by-reg", you can control it from software.
```

Comment:
Because so far "by-reg" is the only possible selection here :)

Reply:
ditto.